contractcourt/breach_arbitrator.go

```diff
@@ -684,8 +686,11 @@ func updateBreachInfo(breachInfo *retributionInfo, spends []spend) (
 // the lingering funds within the channel into the daemon's wallet.
 //
 // NOTE: This MUST be run as a goroutine.
-func (b *BreachArbiter) exactRetribution(confChan *chainntnfs.ConfirmationEvent,
-	breachInfo *retributionInfo) {
+//
+//nolint:funlen
+func (b *BreachArbitrator) exactRetribution(
+	confChan *chainntnfs.ConfirmationEvent, breachInfo *retributionInfo,
+) {
```

Comment:
Why are we inventing new ways to format our function declarations? Can we please stick with the ones defined [here](https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md) or update the doc and have a discussion about it in the PR that updates it?

Reply:
Because I'm still learning them (they aren't intuitive) and the linter didn't catch it. Honest mistake.

Reply:
Fair enough. I guess it didn't help that we [suddenly started using a new format with the Taproot Channel PRs](https://github.com/lightningnetwork/lnd/blob/ce93b236aacdcb3f810c29fff3833228d74dea3c/input/musig2.go#L444) that IMO is [explicitly documented as being wrong in the formatting guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_formatting_rules.md#wrapping-long-function-definitions). Still trying to convince @Roasbeef to roll that back...